pkg/linear-pools/contracts/midas-linear-pool/MidasLinearPoolFactory.sol

```diff
@@ -125,7 +124,7 @@ contract MidasLinearPoolFactory is
 
         bytes memory rebalancerCreationCode = abi.encodePacked(
             type(MidasLinearPoolRebalancer).creationCode,
-            abi.encode(getVault(), _queries)
+            abi.encode(getVault(), _queries, address(mainToken), address(wrappedToken))
```

Comment:
The `Rebalancer` already has direct access to `_mainToken` and `_wrappedToken`, so we shouldn't need to pass these into the constructor.